packages/cli/e2e/deploy.test.ts

```diff
@@ -1,22 +1,28 @@
-import { ERROR_MESSAGES } from "../constants/constants";
 import { prepareEnvironment } from "@gmrchk/cli-testing-library";
 import { copyFile } from "fs/promises";
 import { resolve } from "path";
+import { ERROR_MESSAGES } from "../constants/constants";
 
 // this creates an app, can take some time that's fine
 jest.setTimeout(120_000);
 
 describe("npx thirdweb deploy", () => {
   it("should return deploy page url", async () => {
     const { spawn, cleanup, exists, path } = await prepareEnvironment();
+    // eslint-disable-next-line turbo/no-undeclared-env-vars
```

Comment:
any way to make turbo happy instead of the ignore?

Reply:
Not without passing in the variable into that config, which we want to keep private.